bears/vcs/git/GitCommitBear.py

```diff
@@ -24,4 +25,27 @@ def get_remotes():
 
     def get_head_commit(self):
         with change_directory(self.get_config_dir() or os.getcwd()):
+            command = self.github_merge_commit()
+            if command:
+                return run_shell_command(command)
+
             return run_shell_command('git log -1 --pretty=%B')
+
+    def github_merge_commit(self):
```

Comment:
per the cEP, we're using 'Travis merge commit' as the name of this thing.  Not 'GitHub merge commit'.  Also, as it isnt 100% clear what this 'thing' is, we should not expose this name as part of the public API.  So it should be prefixed with `_`.

Also there should be a docstring here which explains what this thing is.
A link to https://github.com/travis-ci/travis-ci/issues/8400 is the best 'evidence' of this thing being real and known outside of coala.